jenkins/server.sls

```diff
@@ -1,6 +1,7 @@
 {% from "jenkins/map.jinja" import jenkins, deploy with context %}
 include:
   - .repo
+  - .sudoersd
```

Comment:
Please don't include this by default - we don't want all (or even most) of our jenkins to have sudo rights.